src/test/java/domain/CalculatorTest.java

```diff
@@ -0,0 +1,53 @@
+package domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+public class CalculatorTest {
+    private Calculator calculator;
+
+    @BeforeEach
+    void setUp(){
+        calculator = new Calculator();
+    }
+
+    @ParameterizedTest
+    @DisplayName("계산이 올바른지")
+    @CsvSource(value = {"1 + 2 + 3 / 6:1", "11 - 2 * 5 / 1:45", "20 / 4 - 3 - 10:-8"}, delimiter = ':')
+    void calculateTest(String input, double expected) {
+        double actual =  calculator.calculator(input);
+        assertThat(actual).isEqualTo(expected);
+    }
+
+    @ParameterizedTest
+    @DisplayName("문자열이 홀수 개가 아닐 때")
+    @CsvSource(value = {"1 + 2 + 3 / 6 -", "11 - 2 * 5 / 1 1", "20 / 4 - 3 - 10 0"})
+    void strangeArrays(String input) {
+        assertThatExceptionOfType(IllegalArgumentException.class)
+                .isThrownBy(()->
```

Comment:
```suggestion
                .isThrownBy(() ->
```

컨벤션 위반!!!
코드를 작성한 이후 ide의 코드 포맷팅 기능을 사용하여 코드 포맷을 맞추는 것을 추천드려요!!
[네이버 핵데이 Java 코딩 컨벤션 - intellij 설정](https://naver.github.io/hackday-conventions-java/#_intellij)을 확인해보고 적용해보는 것도 좋겠네요 👍